workshop1/03-connecting-the-api.md

```diff
@@ -76,11 +76,12 @@ the `Pet` interface you created in the previous section into its own module.
    look something like this:
 
    ```tsx
-   export default interface Pet {
-       readonly id: string;
-       name: string;
-       description: string;
-   };
+    interface Pet {
```

Comment:
As we chatted about, I think the error you encountered here was from the moment in time before you add any `export` declarations to the file at all, so I think we can leave this as-is.

Reply:
Reverted to the previous definition, thanks for spotting this!